propose_migration.sh

```diff
@@ -13,17 +13,26 @@ if [ "${CIRCLECI-false}" = "true" ]; then
     createdb -U postgres webservice_test_proposed || true
     rm dockstore-webservice/target/detected-migrations.xml || true
 else
+    ## uncomment/comment this section if database is running locally on postgres
     sudo -i -u postgres dropdb webservice_test || true
     sudo -i -u postgres dropdb webservice_test_proposed || true
     sudo -i -u postgres createdb webservice_test || true
     sudo -i -u postgres createdb webservice_test_proposed || true
     rm dockstore-webservice/target/detected-migrations.xml || true
+    ## uncomment/comment this section if running on docker
+#    docker exec -it -u postgres -e PGPASSWORD=dockstore postgres1 dropdb webservice_test || true
+#    docker exec -it -u postgres -e PGPASSWORD=dockstore postgres1 dropdb webservice_test_proposed || true
+#    docker exec -it -u postgres -e PGPASSWORD=dockstore postgres1 psql -c "create database webservice_test with owner = dockstore;" || true
+#    docker exec -it -u postgres -e PGPASSWORD=dockstore postgres1 psql -c "create database webservice_test_proposed with owner = dockstore;" || true
+#    rm dockstore-webservice/target/detected-migrations.xml || true
 fi
 
 ## load up the old database based on current migration
 rm dockstore-webservice/target/dockstore-webservice-*sources.jar || true
 java -jar dockstore-webservice/target/dockstore-webservice-*.jar db migrate dockstore-integration-testing/src/test/resources/dockstore.yml --include 1.3.0.generated,1.4.0,1.5.0,1.6.0,1.7.0,1.8.0,1.9.0,1.10.0
+
 ## create the new database based on JPA (ugly, should really create a proper dw command if this works)
+## remove timeout for mac devices, will have to break manually
```

Comment:
Or install and use [gtimeout](https://stackoverflow.com/questions/3504945/timeout-command-on-mac-os-x). But probably not worth getting into here.